.eslintrc.json

```diff
@@ -13,7 +13,8 @@
   },
   "rules": {
     "react/prop-types": 0,
-    "@typescript-eslint/explicit-function-return-type": 0
+    "@typescript-eslint/explicit-function-return-type": 0,
+    "@typescript-eslint/explicit-module-boundary-types": 0,
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md

Although not required to disable this rule, since it only shows a warning, have done so as compiler should implicitly infer the types, and we have so many methods without explicity typing that it makes it difficult to fully implement.